### DISCUSSION TITLE
Retrieving Domain from deleted project within the same Team

### DISCUSSION THREAD

**Discussion by IAmStevenHale** [vercel/community#6475] [Account Issue]
### Question

I recently deleted a project in Vercel that had a domain attached to it bought and setup via GoDaddy. I then created a new project and went to use the Domain but an error was displayed with the need to update my DNS records with the following:

type: TXT
name: _vercel
value: vc-domain-verify=www.formflowapp.com,1234567890somenumbers

I have done that and it is working as expected. Is this a permanent solution and is there anything else that I need to do?

**Comment by versecafe** [comment]
@IAmStevenHale if your domain is on Vercel's name servers you can use it across any project in the given organization

<img width="1241" alt="image" src="https://github.com/vercel/community/assets/147033096/95f7848b-769b-41fd-b1df-216d48a36871">
<img width="1203" alt="image" src="https://github.com/vercel/community/assets/147033096/339339b5-01f8-4378-a3f0-17dd619c99bc">

and then when you come into a specific project you can use the domains without any additional configuration, you can even use `sub-one.domain.com` on one project and `sub-two.domain.com` in another.

<img width="1203" alt="image" src="https://github.com/vercel/community/assets/147033096/6b698411-637b-44d8-be8b-2aed7dc9eb2d">

**Comment by IAmStevenHale** [comment]
Hey @versecafe 
Thanks for the quick response. The Domain is on GoDaddy and the issue was more that it was associated with a deleted project (I didn't remove the domain before deleting) and was giving errors/warnings about already being used when trying to apply it to a new project.

But in saying that, from your response I see the benefits of transferring the domain to Vercel's name servers and will most likely do that.

Thanks!
